test/routes/paymentCalculation.integration.test.js

```diff
@@ -194,6 +194,7 @@ describe('POST /parcels/{parcelRef/actions/{actionId}/payment-calculation', () =
     })
   })
 
+  // Skip area rule tests because these rules are not yet implemented in new rules engine
```

Comment:
We don't look to be skipping any tests - is this an old comment?

Reply:
😂 Yep, that's from before we updated the new rules engine to cover area rules.

Reply:
And that's now removed. This is why I try so hard not to put comments in code. They always get forgotten about...